src/FastExpressionCompiler/TestTools.cs

```diff
@@ -27,10 +30,10 @@ namespace FastExpressionCompiler;
 [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode", Justification = "This is used for the testing purposes only.")]
 public static class TestTools
 {
-    public static bool AllowPrintIL = false;
-    public static bool AllowPrintCS = false;
-    public static bool AllowPrintExpression = false;
-    public static bool DisableAssertOpCodes = false;
+    public static bool AllowPrintIL;
```

Comment:
These were valid warnings appearing in my build. Though doesn't matter much in this case.